pages/docs/tutorials/create-asyncapi-document.md

```diff
@@ -0,0 +1,221 @@
+---
+title: Create AsyncAPI document
+description: In this tutorial, you'll learn how to create an AsyncAPI document.
+weight: 80
+---
+
+## Introduction
+
+In this tutorial, you'll learn how to create an AsyncAPI document based on a sample real-world use case. Additionally, you will learn about event-driven architecture, message brokers, pub/sub pattern.
+
+Let's pretend you have a company called Smarty Lighting, and you install smart-city streetlight lighting systems. This smart lighting system is a use case of the Internet of Things (IoT). You will create a Smarty Lighting Streetlights API using Node.js and Mosquitto (MQTT) as the message broker. This API will allow you to manage city lights remotely.
+
+You want to build a system that can turn streetlights on and off based on their environmental conditions:
+
+- You will implement an event-driven architecture (EDA) with a message broker in its "center."
+
+- Streetlights will send information about their environmental lighting to the broker.
```

Comment:
```suggestion
- Streetlights will publish information about their environmental lighting to the broker.
```
How about we keep it this way because of how it is mentioned earlier:
> architecture, message brokers, pub/sub pattern.